packages/react/src/Navigate.js

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react";
+import { useRouter } from "./useRouter.js";
+
+export const Navigate = (props) => {
+  const { navigate } = useRouter();
+
+  useEffect(() => {
+    navigate(props);
+  }, [navigate, props]);
```

Comment:
I'm afraid that this will keep firing the effect every time this is rendered, because props isn't the same object - do we know that?

Reply:
I think it is probably better to just have:
```suggestion
  }, [navigate]);
```
and disable the eslint check.